pisces/config.py.deploy

```diff
@@ -1,6 +1,6 @@
 DJANGO_CRON_LOCKFILE_PATH = "${DJANGO_CRON_LOCKFILE_PATH}"
 DJANGO_DEBUG = ${DJANGO_DEBUG}
-DJANGO_PORT = ${DJANGO_PORT}
+APPLICATION_PORT = ${APPLICATION_PORT}
```

Comment:
Does the application actually need this variable? Is it used anywhere in Django?